### PR TITLE
Bump version to v0.16.0.

### PR DESCRIPTION
**Pull request by mzr1996**
Bump version, and benchmark regression will be completed later. Please check it.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+### Highlights
+
+- We have improved compatibility with downstream repositories like MMDetection and MMSegmentation. We will add some examples about how to use our backbones in MMDetection.
+- Add RepVGG backbone and checkpoints. Welcome to use it!
+- Add timm backbones wrapper, now you can simply use backbones of pytorch-image-models in MMClassification!
+
+### New Features
+
+- Add RepVGG backbone and checkpoints. ([#414](https://github.com/open-mmlab/mmclassification/pull/414))
+- timm backbones wrapper. ([#427](https://github.com/open-mmlab/mmclassification/pull/427))
```
Comment (Ezra-Yu) [inline docs/changelog.md:14]: Add timm xxxxxxx.

File: docs/changelog.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+
+### Improvements
+
+- Fix TnT compatibility and verbose warning. ([#436](https://github.com/open-mmlab/mmclassification/pull/436))
+- Support setting `--out-items` in `tools/test.py`.  ([#437](https://github.com/open-mmlab/mmclassification/pull/437))
+- Add datetime info and saving model using torch<1.6 format. ([#439](https://github.com/open-mmlab/mmclassification/pull/439))
+- Improve downstream repositories compatibility. ([#421](https://github.com/open-mmlab/mmclassification/pull/421))
+- Rename the option `--options` in some tools to `--cfg-options`. ([#425](https://github.com/open-mmlab/mmclassification/pull/425))
```
Comment (Ezra-Yu) [inline docs/changelog.md:22]: Rename the option `--options` to `--cfg-options` in some tools.
maybe better

File: docs/changelog.md [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+- Add PyTorch 1.9 and Python 3.9 build workflow, and remove some CI. ([#422](https://github.com/open-mmlab/mmclassification/pull/422))
+
+### Bug Fixes
+
+- Fix format error in `test.py` when metric returns `np.ndarray`. ([#441](https://github.com/open-mmlab/mmclassification/pull/441))
+- Fix `publish_model` bug if no parent of `out_file`. ([#463](https://github.com/open-mmlab/mmclassification/pull/463))
+- Fix num_classes bug in pytorch2onnx.py. ([#458](https://github.com/open-mmlab/mmclassification/pull/458))
+- Fix missing runtime requirement `packaging`. ([#459](https://github.com/open-mmlab/mmclassification/pull/459))
+- Fix simplified model saving in ONNX export tool. ([#438](https://github.com/open-mmlab/mmclassification/pull/438))
```
Comment (Ezra-Yu) [inline docs/changelog.md:31]: Fix saving simplified model bug in ONNX export tool.

File: docs/changelog.md [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
+
+### Docs Update
+
+- Update `getting_started.md` and `install.md` and rewrite `finetune.md`. ([#466](https://github.com/open-mmlab/mmclassification/pull/466))
```
Comment (Ezra-Yu) [inline docs/changelog.md:35]: Update `getting_started.md` and `install.md`, rewrite `finetune.md`.

Comment (mzr1996) [reply docs/changelog.md]: done

File: docs/changelog.md (hunk 3 above)
Comment (mzr1996) [reply docs/changelog.md]: done

**Review by Ezra-Yu — APPROVED** [review]
LGTM.

**Review by Ezra-Yu — APPROVED** [review]
LGTM,

**Pull request merged**